frontend/packages/kubevirt-plugin/integration-tests/tests/wizard.validation.scenario.ts

```diff
@@ -0,0 +1,93 @@
+import { execSync } from 'child_process';
+import { browser } from 'protractor';
+import { click, waitForStringInElement } from '@console/shared/src/test-utils/utils';
+import {
+  KUBEVIRT_TEMPLATES_PATH,
+  KEBAP_ACTION,
+  NOT_RECOMMENDED,
+  PAGE_LOAD_TIMEOUT_SECS,
+  NOT_RECOMMENDED_BUS_TYPE_WARN,
+} from './utils/consts';
+import {
+  Flavor,
+  OperatingSystem,
+  ProvisionConfigName,
+  WorkloadProfile,
+} from './utils/constants/wizard';
+import { Wizard } from './models/wizard';
+import { FlavorConfig } from './utils/types';
+import { customFlavorMemoryHintBlock, clickKebabAction, diskWarning } from '../views/wizard.view';
+import { getProvisionConfigs } from './vm.wizard.configs';
+import { getRandStr } from './utils/utils';
+import { diskInterfaceHelper } from '../views/dialogs/diskDialog.view';
+import { DiskDialog } from './dialogs/diskDialog';
+import { saveButton } from '../views/kubevirtDetailView.view';
+
+describe('Wizard validation', () => {
+  const wizard = new Wizard();
+  const diskDialog = new DiskDialog();
+  const provisionConfig = getProvisionConfigs().get(ProvisionConfigName.CONTAINER);
+  const customFlavorNotEnoughMemory: FlavorConfig = {
+    flavor: Flavor.CUSTOM,
+    cpu: '1',
+    memory: '1',
+  };
+  const customFlavorSufficientMemory: FlavorConfig = {
+    flavor: Flavor.CUSTOM,
+    cpu: '1',
+    memory: '5',
+  };
+
+  beforeAll(async () => {
+    execSync(`kubectl create -f ${KUBEVIRT_TEMPLATES_PATH}/validationCommonTemplate.yaml`);
+  });
+
+  afterAll(() => {
+    execSync(`kubectl delete -f ${KUBEVIRT_TEMPLATES_PATH}/validationCommonTemplate.yaml`);
+  });
+
+  beforeEach(async () => {
+    await wizard.openWizard();
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  it('Displays error when not enough memory provided', async () => {
+    await wizard.selectOperatingSystem(OperatingSystem.VALIDATION_TEST);
+    await wizard.selectFlavor(customFlavorNotEnoughMemory);
+    await browser.wait(
+      waitForStringInElement(customFlavorMemoryHintBlock, 'Memory must be at least'),
+    );
+    await wizard.selectFlavor(customFlavorSufficientMemory);
+    expect(customFlavorMemoryHintBlock.isPresent()).toBe(false);
+  });
+
+  it('Displays warning when interface not recommended', async () => {
+    await wizard.selectProvisionSource(provisionConfig.provision);
+    await wizard.selectOperatingSystem(OperatingSystem.WINDOWS_10);
+    await wizard.selectFlavor(customFlavorSufficientMemory);
+    await wizard.selectWorkloadProfile(WorkloadProfile.DESKTOP);
+    await wizard.fillName(getRandStr(5));
+    await wizard.next();
+    // Network tab
+    await wizard.next();
+    // Storage tab
+
```

Comment:
Here you should see the warning at the disk table before entering the disk-modal, because you are using a template that its rootdisk bus is `SATA` and `SATA` is not on the recommended list. so I think you should move lines 87-91 to here. 
```

Reply:
That is correct, however, it is currently not the case due to 1803132. I have added a TODO comment and will address it in when the bug is resolved.